.evergreen/evergreen.yml.in

```diff
@@ -1245,46 +1817,49 @@ tasks:
   ###
   # E2E TESTS
   ###
-  <% for (const { executableOsId, compileBuildVariant } of RELEASE_PACKAGE_MATRIX) {
-    for (const mVersion of ['stable', '7.0.x', '6.0.x']) {
-      for (const fipsVariant of ['fips', 'nofips']) {
+  # Tests reuse the same compilation build variant, so we create those variations based on this.
+  <% for (const compileBuildVariant of ['build_linux_x64', 'build_linux_arm64', 'build_darwin', 'build_darwin_arm64', 'build_win32']) { 
```

Comment:
Could we compute these values from `ALL_E2E_TEST_BUILD_VARIANTS` (by filtering for the `isBuildOnly` variants)?

Reply:
did this though not with filtering as I split the two arrays completely